manual/Makefile

```diff
@@ -1,5 +1,5 @@
 all: tools
-	cd manual; ${MAKE} all
+	cd manual; ${MAKE} all && cd ..; ${MAKE} tests
```

Comment:
doesn't it work to have it as a second line with just `${MAKE} tests`?

Reply:
This works better indeed, fixed.